### PR TITLE
 js_task-transportation-on-vacation

### PR DESCRIPTION
**Pull request by AndriiYelieva**
Done

### CODE REVIEW

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,21 @@
...
+
+  const week = 50;
+  const moreDay = 20;
+  const day = 40;
+
+  if (days >= 3 && days <= 6) {
+    return days * day - moreDay;
+  } else if (days >= 7) {
+    return days * day - week;
+  }
```
Comment (IvanFesenko) [inline src/calculateRentalCost.js:43]: Do not use else statement

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Lets make some improvements

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -28,7 +28,21 @@
...
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days === 0) {
+    return 0;
+  }
+
+  const week = 50;
+  const moreDay = 20;
+  const day = 40;
```
Comment (IvanFesenko) [inline src/calculateRentalCost.js:37]: Use better naming condition too.

File: src/calculateRentalCost.js [hunk 3]
```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const weekDiscount = 50;
+  const threeDayDiscont = 20;
+  const dayPrice = 40;
+
+  if (days >= 3 && days <= 6) {
+    return days * dayPrice - threeDayDiscont;
```
Comment (nazarmatsevych) [inline src/calculateRentalCost.js:36]: You cal move this expression `days * dayPrice` to variable

**Review by nazarmatsevych — APPROVED** [review]

**Pull request closed without merging**